docs/dev/existing_data_updates.rst

```diff
@@ -1,14 +1,25 @@
 ===============================================================================
-Annual Updates
+Existing Data Updates
 ===============================================================================
-Much of the data we work with is released in a "final" state annually. We typically
-integrate the new year of data over 2-4 weeks in October since, by that
-time, the final release for the previous year have been published by EIA and FERC. We
-also integrate EIA early release data when available. The ``data_maturity`` field will
-indicate whether the data is final or provisional. To see what data we have available
-for each dataset, click on the links below and look at the "Years Liberated" field.
 
-As of spring 2023 the annual updates include:
+Many of the raw data inputs for PUDL are published on a annual or monthly basis. These
+instructions explain the process for integrating new versions of existing data into
+PUDL.
+
+We update EIA monthly data and EPA CEMS hourly data on a quarterly basis.
+
+EIA typically publishes an "early release" version of their annual data in the summer
+followed by a final release in the fall. Our ``data_maturity`` column indicates
+which version has been integrated into PUDL.
```

Comment:
the `data_maturity` column also indicates whether the data is coming from the monthly or otherwise YTD data. it might be good to add that here in well bc it sounds like this is just an indicator or ER or final